src/include/add-on/add-on-workflow.rb

```diff
@@ -1297,22 +1299,13 @@ def GetRepoInfo(this_product, all_products)
     end
 
     def GetAllProductsInfo
-      all_products = Pkg.ResolvableProperties("", :product, "")
+      all_products = Y2Packager::Resolvable.find(:product)
 
       all_products = Builtins.maplist(all_products) do |one_product|
         # otherwise it fills the log too much
-        Builtins.foreach(["license", "description"]) do |key|
-          if Builtins.haskey(one_product, key)
-            Ops.set(
-              one_product,
-              key,
-              Ops.add(
-                Builtins.substring(Ops.get_string(one_product, key, ""), 0, 40),
-                "..."
-              )
-            )
-          end
-        end
+        one_product.license = one_product.license[0..40] + "..."
+        one_product.description = one_product.description[0..40] + "..."
```

Comment:
Um, Rails have a nice [truncate](https://www.rubydoc.info/docs/rails/3.2.8/String:truncate) builtin just for this. Unfortunately we cannot use it...

Reply:
Actually, this truncate should be done by the "presenter" instead of modifying the data here. Hopefully the string is not shared with any other part of the code.

For the time being, let's keep as it is and do not overcomplicate things.